src/app/features/genes/components/gene-evidence-rna/gene-evidence-rna.component.ts

```diff
@@ -194,9 +193,10 @@ export class GeneEvidenceRnaComponent implements AfterViewInit {
   }
 
   onStatisticalModelChange(event: any) {
-    if (!this._gene?.rna_differential_expression) {
+    if (!event)
```

Comment:
original source of error in PR.  Updated method to be more defensive about inputs